### PR TITLE
feat: App-specific docs

### PR DESCRIPTION
**Pull request by dogmar**
## Summary

Shows app-specific documentation on cluster app pages.

## Test Plan

Tested manually

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: www/src/components/app/App.tsx [hunk 1]
```diff
@@ -40,14 +99,19 @@ function AppInternal() {
...
     CLUSTERS_ROOT_CRUMB,
     { label: `${cluster?.name}`, url: `/clusters/${clusterId}` },
     { label: `${name}`, url: `/apps/${clusterId}/${name}` },
   ],
   [cluster?.name, clusterId, name])
 
+  const docs = useMemo(() => getDocsData(data?.repository?.docs),
+    [data?.repository?.docs])
+
+  console.log('docs', data?.repository?.docs)
```
Comment (michaeljguarino) [inline www/src/components/app/App.tsx:112]: rm

File: www/src/models/repo.ts [hunk 2]
```diff
@@ -16,6 +22,9 @@ export const RepoFragment = gql`
     description
     releaseStatus
     documentation
+    docs {
```
Comment (michaeljguarino) [inline www/src/models/repo.ts:25]: Is this going to be sideloaded in repo list queries?  It's far too heavy to be done in those contexts if so

Comment (dogmar) [reply www/src/models/repo.ts:25]: Okay, I'll move to separate query for app pages only.

**Review by maciaszczykm — APPROVED** [review]

**Review by michaeljguarino — APPROVED** [review]
minor fix but otherwise looks good

File: www/src/models/repo.ts [hunk 3]
```diff
@@ -26,6 +32,7 @@ export const RepoFragment = gql`
     publisher { ...PublisherFragment }
   }
   ${PublisherFragment}
+  ${FileContentFragment}
```
[the remark this answers is not in the document]
Comment (dogmar) [reply www/src/models/repo.ts]: Gone.

**Pull request merged**